README.md

```diff
@@ -1 +0,0 @@
-# autowarden-cli
```

Comment:
Why we want to remove our `README.md` file? It will be usefull to make main page of documentation to our project.
Please revert this change (add README.md back).